src/controllers/messagesController.ts

```diff
@@ -101,11 +107,13 @@ export default class MessagesController {
     withUserFromRequest(req, user =>
       tryCatch(
         () =>
-          this.messagesService.getLegalMessage(
-            user,
-            req.params.id,
-            this.tokenService.getPecServerTokenHandler(user.fiscal_code)
-          ),
+          this.messageServicSelector
+            .getOldMessageService()
```

Comment:
may you add a comment on why we don't use `select()` here?

Reply:
Done in [f0c9eca](https://github.com/pagopa/io-backend/pull/882/commits/f0c9ecac61390c067788b47867ae612207dbd68c)